src/main/java/seedu/address/logic/commands/DeleteBySerialNumberCommand.java

```diff
@@ -0,0 +1,80 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.book.Book;
+import seedu.address.model.book.SerialNumber;
+
+/**
+ * Deletes a book identified using it's displayed index from the catalog.
```

Comment:
If I'm not wrong, I think you meant serial number instead of index.

Reply:
I think it's index that is displayed? Just like the origin Address book. So like the whole list of books is displayed and if you want to delete the first book displayed the index is 1. Is it?

Reply:
Yeo tong is right, I shall change the description